components/Withdraw.tsx

```diff
@@ -152,8 +171,8 @@ const Withdraw: React.FC<{}> = () => {
         s,
       },
       {
-        gasLimit: 500000,
         value: verificationFee,
+        gasLimit: 150000,
```

Comment:
at this point with the proof available it should use the value from `estimateGas` + some buffer say `2%`